reliability-engineering/pipelines/tasks/asg-scale-max-capacity.yml

```diff
@@ -0,0 +1,61 @@
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: govsvc/task-toolbox
+    tag: latest
+params:
+  AWS_DEFAULT_REGION: eu-west-2
+  ASG_PREFIX:
```

Comment:
Is it fine to have empty ASG_PREFIX value?

Reply:
yeah it will just fail, having an empty param makes it easier to see what params the tasks expects... the params get set from the the point where the task is used in the Job/plan.